mkat_fpga_tests/test_cbf.py

```diff
@@ -521,20 +521,32 @@ def plot_and_save(freqs, actual_data, expected_data, plot_filename, show=False):
         actual_phases = get_actual_phases()
         expected_phases = get_expected_phases()
         for i, delay in enumerate(test_delays):
-            delta_actual = np.max(actual_phases[i][1]) - np.min(actual_phases[i][1])
-            delta_expected = np.max(expected_phases[i][1]) - np.min(expected_phases[i][1])
-            print "delay: {}ns, expected phase delta: {}, actual_phase_delta: {}".format(
-                delay*1e9, delta_expected, delta_actual)
+            delta_actual = np.max(actual_phases[i][1]) - np.min(
+                actual_phases[i][1])
```

Comment:
Was this not just for debugging? In that case perhaps make it a debug level log. If it is something we want to include in the report instead you should use Aqf...